client/components/helpCentre/KnownIssues.tsx

```diff
@@ -12,26 +12,45 @@ import { gridBase, gridItemPlacement } from '../../styles/grid';
 import { allProductsDetailFetcher } from '../../utilities/productUtils';
 import { ErrorIcon } from '../mma/shared/assets/ErrorIcon';
 
-interface Issue {
-	date: string;
+/*
```

Comment:
I'm OK with commenting this out - but I think we should maybe put a ticket into the health board to take a look at this at a later date.